examples/misc/plotfile_demo_sgskip.py

```diff
@@ -0,0 +1,89 @@
+"""
+=============
+plotfile demo
+=============
+
+Replacing the deprecated `plotfile` by pandas or other matplotlib plotting
```

Comment:
@anntzer This is really a bit difficult to follow. At first glance I was just seeing `plotfile` all over the place. Are you happy with me rewriting the example to better explain/structure it?

Reply:
@timhoffm go for it.